lib/transforms/type_legalizer.cc

```diff
@@ -528,6 +529,11 @@ static void RunOnCommonReductionInstruction(Instruction* inst,
     dt = DataType::INT32;
   }
 
+  constexpr bool is_arg_inst =
+      std::is_same<T, ArgmaxInst>() || std::is_same<T, ArgminInst>();
+  if constexpr (!is_arg_inst) { // NOLINT
+    inst->SetAxis(axis);
+  }
```

Comment:
why we special process Argmax/Argmin

Reply:
For most reductions, axes attribute is a vector. But for argmin/argmax, axis attribute is a scalar

Reply:
ok